### DISCUSSION TITLE
Website is only delivering ICNS

### DISCUSSION THREAD

**Discussion by tsuliwaensis** [elrumo/macOS_Big_Sur_icons_replacements#1518] [Help with the website]
Searching for iOS icons and downloading it sends a `icns` file instead of a `png`.

**Comment by tson018** [comment]
.icns files are composed of different sized .png files. You can open them in Preview and then save the specific .png size you want, although Mac apps use .icns files by default.

**Reply by gotzlotz** [reply]
But the MacOS icon mask is applied. I need the original square version so it can be used for iOS apps.
